src/cmd_definitions.c

```diff
@@ -47,6 +48,17 @@ void cmd_help(char **arg_list, size_t arg_count, char *res_msg) {
 
 }
 
+
+void cmd_kill(char **arg_list, size_t arg_count, char* res_msg) {
+  if (!arg_count) {
+    strcpy(res_msg, "Invalid argument number!");
+  } else if (!sceAppMgrDestroyAppByName(arg_list[1])) {
+    strcpy(res_msg, "App killed sucessfully!");
+  } else {
+    strcpy(res_msg, "Unable to kill App");
+  }
```

Comment:
```suggestion
  if (sceAppMgrDestroyAppByName(arg_list[1]) < 0) {
    strcpy(res_msg, "Error: cannot kill the app. Is the TITLEID correct?\n");
  } else {
    strcpy(res_msg, "Killed.\n");
  }
```

- Sony's convention is `< 0` for errors. `> 0` are not considered errors.\
- You forgot the newline characters.